daemon/.depend

```diff
@@ -0,0 +1,262 @@
+appinfo.o:\
```

Comment:
It is generated automatically. Generating needs to be triggered manually ("make depend"). Why? Because it is useful only for development time incremental builds. Making it fully automated (also during obs builds etc) would accomplish nothing but would require extra build requirements and slow things down a bit.

Reply:
Well... We could then include *.o in the git. ;-)
Having pregenerated file in the git is error prone. Because somebody could forget to update the file when a new source file is added. Also each time you need to update the file which would make commits larger.
I couldn't this file as a pure technical which can be generated automatically during the build time.

Reply:
So, we disagree, you're right but you are wrong, etc ;-)

I'm not going to change this here and now unless somebody insists.